src/sentry/static/sentry/app/components/heroIcon.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+
+const HeroIcon = styled(props => {
+  let {size = 72, ...otherProps} = props;
+  return <InlineSvg size={size} {...otherProps} />;
+})`
```

Comment:
nit: but I would write this

```
const HeroIcon = styled(props => (
  <InlineSvg {...props} />;
)`

HeroIcon.defaultProps = {
  size: '72px'
}
```

Also to note, [`size` needs to be a string](https://github.com/getsentry/sentry/blob/d94c94a6769b8a186f2e1b924e249d4653f56348/src/sentry/static/sentry/app/components/inlineSvg.jsx#L22).